assets/shaders/tex_4xbrz.csh

```diff
@@ -62,20 +63,31 @@ void applyScaling(uvec2 origxy) {
 	//                       17|04|03|02|11
 	//                         |15|14|13|
 
+	uint v[9];
+	v[0] = readInputu(t3.yw);
+	v[1] = readInputu(t3.zw);
+	v[2] = readInputu(t4.zw);
+	v[3] = readInputu(t4.yw);
+	v[4] = readInputu(t4.xw);
+	v[5] = readInputu(t3.xw);
+	v[6] = readInputu(t2.xw);
+	v[7] = readInputu(t2.yw);
+	v[8] = readInputu(t2.zw);
```

Comment:
For the `v[0] == v[1]` type comparisons, do you think it's important that this behaves differently from before with respect to alpha?

Maybe it would actually be better.  Not sure.

-[Unknown]

Reply:
Oh you're right, I missed that - probably best to preserve behavior by masking off some bits, and then try without it but need to set up some way to do proper comparisons for that.